mlflow/entities/assessment_source.py

```diff
@@ -15,14 +15,58 @@ class AssessmentSource(_MlflowObject):
     """
     Source of an assessment (human, LLM as a judge with GPT-4, etc).
 
+    When recording an assessment, it is highly recommended to provide a sufficient
```

Comment:
since it's not just recommended, shall we change the wording here?
```suggestion
    When recording an assessment, it is mandatory to provide a sufficient
```